javascript/javascript1/week2/homework/student-manager.js

```diff
@@ -0,0 +1,44 @@
+const class07Students = [];
+function addStudentToClass(studentName) {
+    if (studentName == null || studentName == "") {
+        console.log("Please fill the name correctly");
+    }
+    else if (getNumberOfStudents() >= 6 && studentName != "Queen") {
```

Comment:
```suggestion
    else if (getNumberOfStudents() > 7 && studentName != "Queen") {
```

Reply:
Actually, I think = is necessary here, as the class should hold max 6 students (unless it's the Queen). And at this point, the new student is not yet added, so if the current size is 6, the class is full, and an error should be displayed.